src/Compilers/CSharp/Portable/Lowering/LocalRewriter/LocalRewriter_Call.cs

```diff
@@ -654,7 +654,18 @@ private static ImmutableArray<RefKind> GetEffectiveArgumentRefKinds(ImmutableArr
                 var receiverNamedType = invokedAsExtensionMethod ?
                                         ((MethodSymbol)methodOrIndexer).Parameters[0].Type.TypeSymbol as NamedTypeSymbol :
                                         methodOrIndexer.ContainingType;
-            isComReceiver = (object)receiverNamedType != null && receiverNamedType.IsComImport;
+                isComReceiver = (object)receiverNamedType != null && receiverNamedType.IsComImport;
+            }
+
+            var @params = methodOrIndexer.GetParameters();
+            if (!@params.IsDefaultOrEmpty)
+            {
+                var thisParam = @params[0];
+                var isRefParam = (thisParam.RefKind & RefKind.Ref) != RefKind.None;
```

Comment:
Here you are using a bitwise check but in the other file a simple `==` is being done. Why the difference?

Reply:
There's no real reason for this to not just be `==` as-is, but I am wondering if other RefKinds need to be accounted for when rewriting.

Will try adding a test case for an `in this` extension method.